LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -104,65 +54,100 @@ public void CreateAccounts(int accountCount)
 
                 Console.WriteLine('\n');
```

Comment:
I don't want to see any Console.WriteLine('\n') calls outside dedicated report methods.